code/game/objects/items/devices/education_chips.dm

```diff
@@ -0,0 +1,92 @@
+//This is for electronic versions of learning, for now solely martial arts
+
+/obj/item/clothing/head/AED
+  name = "Automatic Education Device"
+  desc = "A complex assortment of metal, electrodes, and wires. Simply put the headset on, insert an education chip, and turn it on to instantly learn the contents of said chip."
+  icon_state = "electrode_helmet"
+  actions_types = list(/datum/action/item_action/instant_learn)
+  pocket_storage_component_path = /datum/component/storage/concrete/pockets/tiny/AED
+
+/obj/item/clothing/head/AED/item_action_slot_check(slot)
+  if(slot == SLOT_HEAD)
+    return 1
+
+/obj/item/clothing/head/AED/ui_action_click(mob/user, action)
+  educate()
+
+/obj/item/clothing/head/AED/verb/educate()
```

Comment:
Add user parameter and use it instead of usr.

Reply:
idk what that is tbqh

Reply:
the thing inside parentheses? 
so 
`/obj/item/clothing/head/AED/verb/educate(mob/user)`?

Reply:
Yes, usr should only be used in top level Topic calls if possible.

Reply:
How would I do `set src in usr` then? It only allows for usr.

Reply:
I forgot to mention this shouldn't be a verb and that part is unnecessary.

Reply:
think i got what you wanted